### PR TITLE
Added hits & 2D tracks finder

### PR DESCRIPTION
**Pull request by lzambell**

### CODE REVIEW

**Comment by YoannKermaidic** [comment]
Nice update.
I get the following error:
```
  File "lardon.py", line 193, in <module>
    plot.plot_2dview_2dtracks(to_be_shown=True)
  File "/eos/home-y/ykermaid/sw/dune-lardon/plotting/reco_2d.py", line 169, in plot_2dview_2dtracks
    axs = draw_all_tracks(axs, linewidth=1, legend='2D Track')
  File "/eos/home-y/ykermaid/sw/dune-lardon/plotting/reco_2d.py", line 72, in draw_all_tracks
    axs[iview] = draw_tracks(pos=get_2dtracks_pos(iview,sel), 
NameError: name 'get_2dtracks_pos' is not defined
```

**Comment by lzambell** [comment]
With the latest commit, it should work !

**Review by YoannKermaidic — APPROVED** [review]

File: config.py [hunk 1]
```diff
@@ -34,5 +29,9 @@
 channel_map = ""
 broken_channels = []
 view_offset = []
+view_chan_repet = []
 
+drift_length = 0.
+anode_z = 0.
+view_length = []
```
Comment (olantwin) [inline config.py:36]: Do we need to initialise these here with nonsense values? I think it should be sufficient that they are added as attributes to `cf` in `det_spec.py`.

Not sure whether that changes anything for importing them from other modules though...

File: lardon.py [hunk 2]
```diff
@@ -62,11 +66,14 @@
...
+
+ntot = cf.n_tot_channels*cf.n_sample
 """ mask the unused channels """
-dc.mask_daq = dc.alive_chan
+dc.mask_daq = ~dc.alive_chan
+print("currently ", ntot-sum(dc.mask_daq), " signal hits")
 
 """ setup the decoder """
-reader = (read.top_decoder if elec == "top" else read.bot_decoder)(run, sub, args.file)
+reader = (read.top_decoder if elec == "top" else read.bot_decoder)(run, str(sub), args.file)
```
Comment (olantwin) [inline lardon.py:76]: This isn't necessary anymore as of #33

**Review by olantwin — APPROVED** [review]

**Pull request merged**